### PR TITLE
use new type

### PR DESCRIPTION
**Pull request by thisguycodes**
use a combined `[]Link` type, so we can add new parsing options for people to use

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/conslo/webLinks/pull/4" target="_blank">ReviewNinja

File: webLinks.go [hunk 1]
```diff
@@ -103,6 +103,26 @@ type Link struct {
...
 
+// Links represents a group of links. This allows useful parsing on top of
+// groups of links.
+type Links []Link
+
+// Map returns links mapped in relation:link format. Duplicates have
+// undefined behavior, links without a "rel" param are omitted.
+// Links with a "rel" param, but alternative encoding, are stored according to
+// UTF-8 encoding.
+func (l Links) Map() map[string]Link {
```
Comment (thisguycodes) [inline webLinks.go:114]: wat, no tests?! !fix

File: webLinks_test.go [hunk 2]
```diff
@@ -89,6 +89,40 @@ func TestParseLinksParamValue(t *testing.T) {
 	}
 }
 
+func TestParseLinksIntoMap(t *testing.T) {
```
Comment (thisguycodes) [inline webLinks_test.go:92]: :+1:

File: webLinks.go (hunk 1 above)
Comment (thisguycodes) [inline webLinks.go:114]: !fixed

**Comment by thisguycodes** [comment]
:+1:

**Pull request merged**